_sidebar.md

```diff
@@ -22,4 +22,6 @@
   - [Template Pallet](5-develop/3-template-pallet.md)
   - [Sending Messages](5-develop/4-sending-messages.md)
   - [Receiving Messages](5-develop/5-receiving-messages.md)
+- Register a Rococo Parachain
+  - [Register a Rococo Parachain](6-register/1-register.md)
```

Comment:
WHY??? Didn't we say that it will stay out of the index?

Reply:
Agreed, this should not be in the ToC.

The workshop teaches how to do this stuff locally. Registering on the public network is supplementary material.

Reply:
LOL geeze sorry...